app/views/notes/index.html.erb

```diff
@@ -8,8 +9,37 @@
 
 <% if @notes.empty? %>
   <h4><%= t ".no_notes" %></h4>
-
 <% else %>
+  <%= form_tag(url_for("controller" => "notes", "action" => "index"), "method" => :get, "data" => { "turbo" => true, "turbo-frame" => "pagination", "turbo-action" => "advance" }) do %>
+    <div class="row gx-2">
+      <div class="col-md-auto mb-3">
+        <%= label_tag :status, t(".status") %>
+        <%= select_tag  :status,
+                        options_for_select([[t(".all"), "all"], [t(".open"), "open"], [t(".closed"), "closed"]], params[:status]),
+                        :include_blank => t(".select_status"),
```

Comment:
Why do you need `:include_blank`s? You have labels that say what these selects are and the filter acts as if the first options are selected, but you don't show them as selected.

Reply:
Thank you for pointing that out. I've remove them and added preselected all fields accordingly.